Source/SantaGUI/SNTMessageWindowController.h

```diff
@@ -23,13 +24,18 @@
 ///
 ///  Controller for a single message window.
 ///
-@interface SNTMessageWindowController : NSWindowController
+@interface SNTMessageWindowController : NSWindowController<NSApplicationDelegate,
+                                                           WebPolicyDelegate,
+                                                           WebFrameLoadDelegate>
+@property(weak) IBOutlet NSWindow *mainWindow;
+@property(weak) IBOutlet WebView *webView;
+
 
 - (instancetype)initWithEvent:(SNTStoredEvent *)event andMessage:(NSString *)message;
 
 - (IBAction)showWindow:(id)sender;
 - (IBAction)closeWindow:(id)sender;
-- (IBAction)showCertInfo:(id)sender;
+- (IBAction)showCertInfo;
 
 ///  Reference to the "Bundle Hash" label in the XIB. Used to remove if application
```

Comment:
Remove the remaining IBOutlet properties in this header (`bundleHashLabel`, `bundleHashTitle`, `hashingIndicator`, `openEventButton`) as they presumably aren't used anymore.